pkg/cvo/internal/operatorstatus.go

```diff
@@ -48,16 +49,16 @@ func readClusterOperatorV1OrDie(objBytes []byte) *configv1.ClusterOperator {
 }
 
 type clusterOperatorBuilder struct {
-	client   ClusterOperatorsGetter
-	raw      []byte
-	modifier resourcebuilder.MetaV1ObjectModifierFunc
-	mode     resourcebuilder.Mode
+	client       ClusterOperatorsGetter
+	createClient configclientv1.ClusterOperatorInterface
```

Comment:
qq: if this is supposed to be only the Create, is there a create only interface??

Reply:
> qq: if this is supposed to be only the Create, is there a create only interface??

No and we need update status.